lib/script/interpreter.js

```diff
@@ -627,16 +632,21 @@ Interpreter.prototype.step = function () {
 
   function isOpcodeDisabled (opcode) {
     switch (opcode) {
-      case Opcode.OP_INVERT:
       case Opcode.OP_2MUL:
       case Opcode.OP_2DIV:
-      case Opcode.OP_MUL:
-      case Opcode.OP_LSHIFT:
-      case Opcode.OP_RSHIFT:
 
         // Disabled opcodes.
         return true
 
+      case Opcode.OP_INVERT:
+      case Opcode.OP_MUL:
+      case Opcode.OP_LSHIFT:
+      case Opcode.OP_RSHIFT:
+        // Opcodes that have been reenabled.
+        if ((self.flags & Interpreter.SCRIPT_ENABLE_MAGNETIC_OPCODES) === 0) {
```

Comment:
Is this something that should be refactored out and not copied from the legacy BAB code?

Reply:
You might be right. Certainly in another 6 months or 1 year it will be dead code. I will see how the rest of testing goes and I might just remove Magnetic and Monolith flags.